Gruntfile.js

```diff
@@ -29,18 +29,18 @@ module.exports = function(grunt) {
         pkg: packageInfo,
 
         jshint: {
+            options: {
+                jshintrc: '.jshintrc'
+            },
             player : [
-                'src/js/**/*.js'
+                'src/js/{,*/}*.js'
```

Comment:
why this? I'm pretty sure ** is the preferred way for glob https://github.com/isaacs/node-glob